news/2025/2025-05-25-china-lan-2025.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: China LAN 2025
+date: 2024-05-25 12:00:00 +0000
+---
+
+We're excited to announce China LAN 2025, an in-person osu! and osu!mania tournament based in Shanghai!
+
+![](/wiki/shared/news/2025-05-25-china-lan-2025/banner.jpg)
+
+The tournament will feature many custom maps, original songs, and the best players of the Chinese region! This is your chance to prove your skills and meet your favourite players face to face.
+
+**[View the Chinese version of this article here. | 点击此处查看中文版主题。](https://osu.ppy.sh/community/forums/topics/2080306)**
+
+## Where and when?
+
+**China LAN 2025** will take place in **Shanghai, China** and last for 2 days — July 12th and 13th!
+
+The exact **venue location** will be **announced later**, pending final government approval. Details on ticket sales will also be announced in the event's [Discord server](https://discord.com/invite/dpa5Qr4tu8) when available.
```

Comment:
I assume there's no chance of it not getting approval? Kinda weird to announce a RL event with no venue listed though..

Reply:
it's weird yeah, apparently it's basically confirmed and signed but due to regulations they can't announce before official approval

Reply:
We could hold off on this post until closer to the event? It seems a bit too far out to garner attention for the event, if that's their goal.

Or are entries still open?

Reply:
more context in the [osu-news thread](https://discord.com/channels/188630481301012481/1375180746515415040/1376740951686516757)

above solution would require prioritised tournament screening on the support end